src/fdcache_entity.cpp

```diff
@@ -426,7 +426,7 @@ int FdEntity::Open(const headers_t* pmeta, off_t size, time_t time, int flags, A
         // check only file size(do not need to save cfs and time.
         if(0 <= size && pagelist.Size() != size){
             // truncate temporary file size
-            if(-1 == ftruncate(physical_fd, size)){
+            if(-1 == ftruncate(physical_fd, size) || -1 == fsync(physical_fd)){
```

Comment:
Is it necessary to `fsync` locally?

Reply:
Same code has attached it to the code behind this, so I attached it just in case.
It may not be necessary in practice, but since it is only this case and it is a sync to a local file, I think that there is almost no impact on performance.